composer/datasets/ade20k.py

```diff
@@ -112,6 +140,87 @@ def build_ade20k_dataloader(
     )
 
 
+def build_streaming_ade20k_dataloader(
+    batch_size: int,
+    remote: str,
+    *,
+    version: int = 2,
+    local: str = '/tmp/mds-cache/mds-ade20k/',
+    split: str = 'train',
+    drop_last: bool = True,
+    shuffle: bool = True,
+    base_size=512,
+    min_resize_scale: float = 0.5,
+    max_resize_scale: float = 2.0,
+    final_size: int = 512,
+    ignore_background: bool = True,
+    **dataloader_kwargs,
+):
+    """Build an ADE20k streaming dataset.
+
+    Args:
+        batch_size (int): Batch size per device.
+        remote (str): Remote directory (S3 or local filesystem) where dataset is stored.
+        version (int): Which version of streaming to use. Default: ``1``.
```

Comment:
version conflict with line 147. Should be `2`.

Reply:
Fixed!